Formula/ciphey.rb

```diff
@@ -0,0 +1,187 @@
+class Ciphey < Formula
+  include Language::Python::Virtualenv
+
+  desc "Automatic decryption and decoding tool"
+  homepage "https://github.com/Ciphey/Ciphey"
+  url "https://files.pythonhosted.org/packages/a5/db/9e0411803c768cd7f5c6986c9da406ae7e4b6b6a1d8ad0dc191cff6dbdaf/ciphey-5.14.0.tar.gz"
+  sha256 "302a90261e9acc9b56ea29c313192f0c6f6ce112d37f4f9d404915052e19bf09"
+  license "MIT"
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "poetry" => :build
+  depends_on "swig" => :build
+  depends_on "libyaml"
+  depends_on "python@3.9"
+  depends_on "six"
+
+  resource "appdirs" do
+    url "https://files.pythonhosted.org/packages/d7/d8/05696357e0311f5b5c316d7b95f46c669dd9c15aaeecbb48c7d0aeb88c40/appdirs-1.4.4.tar.gz"
+    sha256 "7d5d0167b2b1ba821647616af46a749d1c653740dd0d2415100fe26e27afdf41"
+  end
+
+  resource "astroid" do
+    url "https://files.pythonhosted.org/packages/6a/7f/90312f42efc2a5249f26e436be10f0c53c512b2f3a36af364cd021283660/astroid-2.6.2.tar.gz"
+    sha256 "38b95085e9d92e2ca06cf8b35c12a74fa81da395a6f9e65803742e6509c05892"
+  end
+
+  resource "base58" do
+    url "https://files.pythonhosted.org/packages/b5/c1/8e77d5389cf1ea2535049e5ffaeb241cce21bcc1c42624b3e8d0fb3bb607/base58-2.1.0.tar.gz"
+    sha256 "171a547b4a3c61e1ae3807224a6f7aec75e364c4395e7562649d7335768001a2"
+  end
+
+  resource "base91" do
+    url "https://files.pythonhosted.org/packages/5b/19/b0f1df635561be1bcd74c652a2e611ebf0a9d5f5f10be9d8421733e2cb04/base91-1.0.1.tar.gz"
+    sha256 "5b284a2ba3c97be1eb9473f3af94a9bf141d61005d836e75e645d2798da58799"
+  end
+
+  resource "cipheycore" do
```

Comment:
Perhaps it would be better to separate this from the PyPI resources, that way it can easily be commented out to use `update-python-resources`.